### PR TITLE
feat(select): match on popup with _select in ctx

### PR DESCRIPTION
**Pull request by Guria**
@nik-kor @awinogradov 
This looks like better way to match popup without [extra replacing result of `applyNext`](https://github.com/awinogradov/react-bl/commit/4a19457aa793256ff1b3b154347205ae6fb66968#diff-13eb908534f8bf1a18748b123d578899R8)

### CODE REVIEW

**Comment by nik-kor** [comment]
LGTM

File: src/components/select/__popup/select__popup.bemhtml.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    .def()(function() {
+        const React = require('react');
+        const Popup = require('../src/components/popup/popup');
+        const selectComponent = this._select;
+
+        return React.createElement(Popup, {
+            ref: popup => {
+                selectComponent._select._popup = popup;
+            },
+            directions: ['bottom-left'],
```
Comment (awinogradov) [inline src/components/select/__popup/select__popup.bemhtml.js]: One direction?

**Comment by awinogradov** [comment]
@Guria if you have to pass directions do it, please) But if not, it's LGTM.

**Comment by Guria** [comment]
@awinogradov Don't sure if it was intended so just left as it was @nik-kor implementation. Of course I can pass it.

**Comment by Guria** [comment]
fixed

**Comment by awinogradov** [comment]
Cool!!!

**Comment by awinogradov** [comment]
@Guria thanks!

**Pull request merged**